### ISSUE TITLE
An exception "ValueError: Invalid control character at..." caused by colorized echos.

### ISSUE THREAD

**Issue by knowack1** [rizsotto/Bear#154]
Hello,
That you for the great tool. I hope my report will be useful.
I have faced an exception while using bear.
Exception is caused by colourized output (control characters).
Please find reproduction script/makefile attached:
[bear.zip](https://github.com/rizsotto/Bear/files/950893/bear.zip)

Works fine when strict flag is set to False:
```diff
diff --git a/bear/main.py.in b/bear/main.py.in
index a1d482b..a99c889 100644
--- a/bear/main.py.in
+++ b/bear/main.py.in
@@ -265,7 +265,7 @@ def parse_exec_trace(filename):
 
     logging.debug('parse exec trace file: %s', filename)
     with open(filename, 'r') as handler:
-        entry = json.load(handler)
+        entry = json.load(handler, strict = False)
         return Execution(
             pid=entry['pid'],
             cwd=entry['cwd'],
```

Thank you.

**Comment by rizsotto** [comment]
Thanks for the report @knowack1 ! Will look at it this week. Your suggestion looks good. Although I might want to filter out this at the `libear` side. Will think about it.

**Comment by rizsotto** [comment]
This is fixed in 2.3.3. Please reopen if you find it still failing.
